app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -98,6 +98,21 @@ $after-button-padding-left: govuk-spacing(4);
   border-top: 1px solid $govuk-brand-colour;
 }
 
+
+.gem-c-layout-super-navigation-header--blue-background.gem-c-layout-super-navigation-header--menu-open {
+  // Add a blue section to the bottom of the open menu
```

Comment:
A small thing but we could say why we're doing this in the comment as that might not be clear.
```
// Add a blue section to the bottom of the open menu.
// This increases the space above the blue banner on the homepage when the menu is open for improved readability.
```